strategy/sampling/sampling_rule_test.go

```diff
@@ -343,3 +343,145 @@ func BenchmarkCentralizedRule_Sample(b *testing.B) {
 		}
 	})
 }
+
+func TestCentralizedRuleAppliesTo(t *testing.T) {
+	type testCase struct {
+		testName string
+		rule     *CentralizedRule
+		request  *Request
+		expected bool
+	}
+
+	testCases := []testCase{
+		{
+			testName: "if the request is empty and the rule has a specific ServiceName, AppliesTo should return false",
```

Comment:
Can we add one test case here, where all the values of rule are populated like URLPath, ServiceName, ServiceType, Host and httpMethod. Now we pass the empty request, AppliesTo should return false ?